mica/vv/core.py

```diff
@@ -269,7 +268,7 @@ def _save_info_json(self, file=None):
         logger.info("Saved JSON to {}".format(file))
 
     def slots_to_db(self):
-        if self.info()["aspect_1_id"] is None:
+        if self.info().get("aspect_1_id") is None:
```

Comment:
Why are you using `get` instead of `__getitem__`? Is it that at some point you had this?
```
if self.info().get("aspect_1_id", None) is None
```

Reply:
Just making sure it was on purpose and not left from unrelated edit.

Reply:
Ah - you can see it was on purpose from my commit history but it is a good point that I didn't call this out in the description!